### PR TITLE
Feat/single item page

### PR DESCRIPTION
**Pull request by dbjowett**
- deleted /desktop
- Single item page done 
- Splash screen component finished (needs implementing to show before Onboarding)

### CODE REVIEW

File: apps/web/src/components/DriveLayouts/Background/MainBackground.module.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 .container {
-  position: relative;
+  position: absolute;
   width: 100%;
   height: 100%;
   background: #f8f8f9;
```
Comment (otabeknos1rov) [inline apps/web/src/components/DriveLayouts/Background/MainBackground.module.scss:5]: Can we use colors from root css variables?

File: apps/web/src/components/TitleAndSearch/FilterBar/FilterBar.module.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.MainBySortBy {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  left: 4%;
+  top: 37%;
+  min-width: 200px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/components/TitleAndSearch/FilterBar/FilterBar.module.scss:10]: How it comes to use fonts from _fonts.scss?

File: apps/web/src/components/TitleAndSearch/FilterBar/FilterBar.module.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  }
+}
+
+.MainBySortDet {
+  position: absolute;
+  left: 4%;
+  width: 24%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/components/TitleAndSearch/FilterBar/FilterBar.module.scss:43]: here also

File: apps/web/src/pages/SingleItem/index.tsx [hunk 4]
```diff
@@ -52,67 +45,99 @@ function SingleItem() {
...
-              <div className={styles.TableHeadings}>
-                <h3>Collection </h3>
-                <h3 style={{ color: '#d7d7d7' }}>Buying History</h3>
-              </div>
-
-              {/* Start of Table */}
+              <div className={styles.HeadAndTableContainer}>
+                <div className={styles.TableHeadings}>
+                  <h3>Collection </h3>
+                  <h3 style={{ color: '#d7d7d7' }}>Buying History</h3>
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/index.tsx:76]: can we use instead inline style , classname?

File: apps/web/src/pages/SingleItem/index.tsx [hunk 5]
```diff
@@ -34,15 +23,19 @@ function SingleItem() {
...
+    );
   }
   function TableRows({ rowData }: { rowData: RowData }) {
     return (
       <tr className={styles.RowContainer}>
         <td>{rowData.buyerData}</td>
         <td>{rowData.editionData}</td>
-        <td style={{ color: '#5F93F1' }}>{rowData.dateData}</td>
-        <td style={{ textTransform: 'uppercase', color: '#64C2D1' }}>{rowData.priceData}</td>
+        <td style={{ color: '#a8a8a8', textTransform: 'uppercase', fontSize: '12px' }}>{rowData.dateData}</td>
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/index.tsx:37]: please use classname instead style

File: apps/web/src/components/DriveLayouts/Background/MainBackground.module.scss (hunk 1 above)
Comment (dbjowett) [reply apps/web/src/components/DriveLayouts/Background/MainBackground.module.scss:5]: Oops I think I forgot to add global variables in a few place. I will double check everything.

**Review by JM066 — APPROVED** [review]

**Review by IslomJonMirzakbarov — APPROVED** [review]

**Pull request merged**